packages/eslint-plugin-brikke/lib/rules/no-undeclared-imports.js

```diff
@@ -73,7 +73,9 @@ module.exports = {
       if (isBuiltin(sourceValue)) {
         return
       }
-      if (excludedModules.includes(sourceValue)) {
+      if (
+        excludedModules.some(pattern => sourceValue.includes(pattern))
```

Comment:
```suggestion
        excludedModules.some(pattern => pattern instanceof RegExp ? pattern.test(sourceValue) : sourceValue === pattern)
```

can you try this with:
```
    {
      code: "import foo from '$src/foo'",
      options: [{ ...getOptions(), excludedModules: [/^\$src\//] }],
    }
```

I think it's more flexible with a regexp, wydt?

Reply:
LGTM